### PR TITLE
Generate PDF with change report content

### PR DESCRIPTION
**Pull request by wschaefer-cfa**
- the change report pdf is generated with wicked_pdf
- pdf verifications are attached using CombinePdf
- image verifications are rendered in the html that is converted to a pdf using wicked_pdf

[#160521773]

### CODE REVIEW

File: app/controllers/admin/change_reports_controller.rb [hunk 1]
```diff
@@ -17,5 +17,24 @@ class ChangeReportsController < Admin::ApplicationController
...
+        end
+        format.pdf do
+          change_report = ChangeReport.find(params[:id])
+          pdf_from_html = render_to_string(
+            pdf: "change_report",
+            template: "admin/change_reports/show",
+            locals: { change_report: ChangeReportDecorator.new(change_report) },
+          )
+
+          pdf = PdfBuilder.new(pdf_from_html: pdf_from_html, attachments: change_report.pdf_letters).run
```
Comment (luigi) [inline app/controllers/admin/change_reports_controller.rb]: I'm reading this as the attachments only being the `pdf_letters` (document type `application/pdf`). What happens to the documents uploaded as JPEG files?

File: app/decorators/change_report_decorator.rb [hunk 2]
```diff
@@ -0,0 +1,45 @@
+class ChangeReportDecorator < SimpleDelegator
```
Comment (luigi) [inline app/decorators/change_report_decorator.rb]: I like it!

File: app/controllers/admin/change_reports_controller.rb (hunk 1 above)
Comment (wschaefer-cfa) [reply app/controllers/admin/change_reports_controller.rb]: @luigi the images are rendered into the show.pdf.erb and so they get included in the pdf_from_html. I thought that was the simplest thing to do, but open to changing it if it's confusing?

Comment (luigi) [reply app/controllers/admin/change_reports_controller.rb]: I think that's a perfectly good way to do it! I just missed it when reading through the ERB file. :+1:

**Pull request merged**